addons/frag/CfgAmmo.hpp

```diff
@@ -1,275 +1,36 @@
 #define BASE_DRAG -0.01
 #define HD_MULT 5
 #define BASE_DRAG_HD (BASE_DRAG * HD_MULT)
+#define ACE_FRAG_RM_EH class EventHandlers: EventHandlers {\
+    class ADDON {\
+        init = "";\
+    };\
+}
+#define ACE_FRAG_ADD_EH_BASE class EventHandlers {\
+    class ADDON {\
+        init = QUOTE(if (isServer) then {_this call FUNC(initRound);};);\
+    };\
+}
+#define ACE_FRAG_ADD_EH class EventHandlers: EventHandlers {\
+    class ADDON {\
+        init = QUOTE(if (isServer) then {_this call FUNC(initRound);};);\
+    };\
+}
 
 class CfgAmmo {
 
-    // ~~~~ Bombs:
-    class ammo_Bomb_LaserGuidedBase;
-    class Bo_GBU12_LGB: ammo_Bomb_LaserGuidedBase {
-        GVAR(enabled) = 1;
+    #include "cfgAmmoBaseEH.hpp"
```

Comment:
Change the file to start with a capital letter.
```suggestion
    #include "CfgAmmoBaseEH.hpp"
```

Reply:
Fixed